bin/release-list/src/release_list/main.clj

```diff
@@ -80,7 +80,7 @@
 
 (def command
   "GitHub CLI command to list all releases, exluding RCs"
-  "gh release list --repo metabase/metabase --limit 10000 --exclude-pre-releases --exclude-drafts")
+  "gh release list --repo metabase/metabase --limit 10000 --exclude-pre-releases")
```

Comment:
I think since we're running this right after the draft notes are generated, we need to include drafts, right?